README.md

```diff
@@ -16,4 +16,47 @@ Either the key `a...` is used in the kibana config as this is the default:
 xpack.encryptedSavedObjects.encryptionKey: "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
 ```
 
-The alternative is to use `ES_SAVED_KEY` and pass it to fleet-server during setup with the value of the encryption key used in Kibana.
\ No newline at end of file
+The alternative is to use `ES_SAVED_KEY` and pass it to fleet-server during setup with the value of the encryption key used in Kibana.
+
+
+### Kibana
+
+Currently there is some work to do to be able to run Kibana with Fleet Server and all the features are not yet supported, in the future, these workarounds will not be needed anymore.
+
+* Start fleet-server before Kibana, to create the mappings in ES.
+* Create and use a custom user as the `kibana_system` user
```

Comment:
For development, could I just pass in `elastic` / `changeme` and it would work too?

Reply:
it's not possible to pass `elastic` as the kibana user this will be rejected

Reply:
Good to know.